R/MEND2.R

```diff
@@ -0,0 +1,225 @@
+#' Define 2 carbon pool MEND
```

Comment:
@jianqiuz this is the package adaptation of @jianqiuz  `MENDplus.R`. What are your thoughts on the nomenclature of `MEND2`? Do you have any other ideas? It is a fairly easy fix if we want to change the name.

Reply:
We can create some naming rules